packages/tasks/ios/install.js

```diff
@@ -0,0 +1,43 @@
+const process = require('process');
```

Comment:
Why this file is in `packages/...` rather than `src/tasks/...` ?

Reply:
You're right, this folder shouldn't be there anymore (rebased from the old architecture).